pod/dressing/views.py

```diff
@@ -0,0 +1,149 @@
+"""Esup-Pod dressing views."""
+from django.shortcuts import render, redirect, get_object_or_404
+from pod.main.views import in_maintenance
+from django.urls import reverse
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib.auth.decorators import login_required
+from django.utils.translation import ugettext as _
+from django.views.decorators.csrf import csrf_protect
+from django.contrib import messages
+from django.db.models import Q
+
+from pod.video.models import Video
+from pod.video_encode_transcript.encode import start_encode
+from .forms import DressingForm, DressingDeleteForm
+from .models import Dressing
+
+
+@csrf_protect
+@login_required(redirect_field_name="referrer")
+def video_dressing(request, slug):
+    """View for video dressing"""
+    if in_maintenance():
+        return redirect(reverse("maintenance"))
+    video = get_object_or_404(Video, slug=slug, sites=get_current_site(request))
+
+    user = request.user
+    users_groups = user.owner.accessgroup_set.all()
+    dressings = Dressing.objects.filter(
+        Q(owners=user) | Q(users=user) | Q(allow_to_groups__in=users_groups)).distinct()
+
+    if request.method == 'POST':
+        selected_dressing_value = request.POST.get("selected_dressing_value")
+        if selected_dressing_value is not None:
+            selected_dressing = get_object_or_404(Dressing, pk=selected_dressing_value)
+            existing = Dressing.objects.filter(videos=video)
+            for dressing in existing:
+                dressing.videos.remove(video)
+                dressing.save()
+
+            selected_dressing.videos.add(video)
+            selected_dressing.save()
+            start_encode(video.id)
+        else:
+            start_encode(video.id)
+
+    try:
+        current = Dressing.objects.get(videos=video)
+    except Dressing.DoesNotExist:
+        current = None
+
+    return render(
+        request,
+        "video_dressing.html",
+        {"video": video, "dressings": dressings, "current": current}
+    )
+
+
+@csrf_protect
+@login_required(redirect_field_name="referrer")
+def dressing_edit(request, dressing_id):
+    """Edit a dressing object."""
+    dressing_edit = get_object_or_404(Dressing, id=dressing_id)
+    form_dressing = DressingForm(
+        instance=dressing_edit,
+        is_staff=request.user.is_staff,
+        is_superuser=request.user.is_superuser,
+        user=request.user,
+    )
+
+    if request.method == 'POST':
+        form_dressing = DressingForm(
+            request.POST,
+            instance=dressing_edit,
+            user=request.user,
+        )
+        if form_dressing.is_valid():
+            messages.add_message(request, messages.INFO,
+                                 _("The changes have been saved."))
+            form_dressing.save()
+            return redirect(reverse("dressing:my_dressings"))
+    page_title = _("Edit the dressing") + f' "{dressing_edit.title}"'
```

Comment:
On pourrait mettre le f-string en début : `page_title = f'{_("Edit the dressing")} {dressing_edit.title}'`